### PR TITLE
fix(client): added new faq pay via aws

### PR DESCRIPTION
**Pull request by joyal007**
Draft: #7060

## PR Details

Added new FAQ to purchase page as mentioned in the issue. The link is not yet finalised

### Summary
🆕💳☁️

Add AWS Marketplace question to Purchase FAQ. This change updates the `FAQ.tsx` component to inform potential customers about the possibility and advantages of paying for the subscription through AWS Marketplace.

> _FAQ component_
> _`AWS Marketplace` added_
> _Winter sales may rise_

### Walkthrough
* Add a new FAQ item about AWS Marketplace integration ([link](https://github.com/amplication/amplication/pull/7083/files?diff=unified&w=0#diff-d1632ef604f51b2b472c1c0d4ab70d88de050c5115abef99f064234f3f9831a4R117-R142))

## PR Checklist

- [ ] Tests for the changes have been added
- [X] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

**Comment by yuval-hazaz** [comment]
Hi @Sushrut1101 thank you for your contribution.
Before we review this PR, you must be assigned to work on it.

**Comment by souravjain540** [comment]
@joyal007 please comment on the issue and get it assigned.

File: packages/amplication-client/src/Purchase/FAQ.tsx [hunk 1]
```diff
@@ -113,6 +113,32 @@ export const FAQ = () => {
...
+        <Question
+          question="Can I pay for my subscription through AWS Marketplace?"
+          answer={
+            <div>
+              Yes, absolutely! We understand that many businesses prefer the
+              convenience and security of managing their software subscriptions
+              through AWS Marketplace. That's why we've made it possible for you
+              to subscribe to our services directly from there.
+              <br />
+              <br />
```
Comment (abrl91) [inline packages/amplication-client/src/Purchase/FAQ.tsx:125]: Please avoid using `<br/>`. Use another HTML block element instead

**Review by abrl91 — CHANGES_REQUESTED** [review]

File: packages/amplication-client/src/Purchase/FAQ.tsx (hunk 1 above)
Comment (levivannoort) [reply packages/amplication-client/src/Purchase/FAQ.tsx]: Hey @joyal007 just tried to do a check for quality assurance sake, seems like you're on the the right track, but the <p> elements might cause the bigger space between the divider and the answer:

example:
image

**Comment by overbit** [comment]
@mulygottlieb is this still relevant?

**Comment by overbit** [comment]
@mulygottlieb @PazYanoverr  is this still relevant?

Currently I see this in prod:
image

**Comment by PazYanoverr** [comment]
@overbit - You are right- it's not relevant any more.

**Pull request closed without merging**